templates/books-index.html

```diff
@@ -0,0 +1,33 @@
+<div ng-repeat="book in bookIndexController.books">
+  <div class="col-md-10 col-md-offset-1">
+	<div class="panel panel-default">
```

Comment:
Why do you have all these divs within divs? This smells like copy-pasted code that you never modified or understood.